src/main/java/com/evanlennick/retry4j/CallExecutor.java

```diff
@@ -39,15 +39,39 @@
 
     private Status<T> status = new Status<>();
 
+    /**
+     * Use {@link CallExecutorBuilder} to build {@link CallExecutor}
+     */
+    @Deprecated
     public CallExecutor() {
```

Comment:
same here, do you think we should just remove this instead of deprecating and get it done with?